dev/Kozani/KozaniHostRuntime/main.cpp

```diff
@@ -194,33 +205,38 @@ void GetConfigurationFiles(std::wstring& rdpFilePath, std::wstring& additionalSe
     }
 }
 
+#ifdef WAIT_FOR_DEBUGGER
+void WaitForDebugger()
+{
+    while (!IsDebuggerPresent())
+    {
+        Sleep(3000);
+    }
+}
+#endif
+
 int APIENTRY wWinMain(
     HINSTANCE /*hInstance*/,
     HINSTANCE /*hPrevInstance*/,
-    PWSTR pCmdLine,
+    PWSTR /*pCmdLine*/,
     int /*nCmdShow*/) try
 {
     wil::SetResultLoggingCallback(&TraceFailureFromProvider<Microsoft_Kozani_HostRuntime_TraceLogger>);
     winrt::init_apartment();
 
-    auto activatedEventArgs{ AppInstance::GetActivatedEventArgs() };
-    std::wstring remoteAumid;
-    if (activatedEventArgs.Kind()== winrt::Windows::ApplicationModel::Activation::ActivationKind::Launch)
-    {
-        // For launch activation, we can simply get the AUMID from command line arguments.
-        remoteAumid = GetRemoteAumidFromCommandLineArgs(pCmdLine);
-    }
-    else
-    {
-        remoteAumid = GetRemoteAumidFromManifest();
-    }
+#ifdef WAIT_FOR_DEBUGGER
+    WaitForDebugger();
+#endif
```

Comment:
Debugging code?

Obsolete and should be deleted?

Reply:
Leave it here so it can be enabled to facilitate debugging easily.